CHANGELOG

```diff
@@ -98,240 +98,6 @@ Cleanup/Removal:
 - Deprecated (and likely removed in the next release)
   - XTandemAdapter
 
-- CHANGED PARAMETERS:
```

Comment:
Please mention `CHANGELOG_PARAMS` here, there are no mentions of it anywhere in the repo

Reply:
good point can do - or you just suggest in-line and I accept